content/docs/working-groups/rfc-2229/minutes/2019.03.05-roadmap-plan.md

```diff
@@ -1,6 +1,6 @@
-- We chalked out the detailed [roadmap][roadmap] for the working group.
```

Comment:
If the intent here is to avoid Hugo interpreting the opening lists as metadata, you could probably add the following to the top of affected files:

```
---
---
```

I've not tested this, but I've seen it work w/ other static site generators.